public/locales/en/common.json

```diff
@@ -7,6 +7,50 @@
   "developer": "Developer",
   "about": "About",
   "language": "中文",
-  "lets-talk": "Contact With Us",
-  "login": "Login"
+  "lets-talk": "Appointment",
+  "login": "Login",
+  "footer-menu-1-title": "Products",
+  "footer-menu-1-1-title": "Large Scale Customer Acquisition",
+  "footer-menu-1-2-title": "Marketing Automation",
+  "footer-menu-1-3-title": "Customer Conversation Panel",
+  "footer-menu-1-4-title": "Data Panel",
+  "footer-menu-1-5-title": "Management Center",
+  "footer-menu-1-6-title": "Security Compliance",
+  "footer-menu-1-7-title": "Cloud Native",
+  "footer-menu-1-8-title": "RPA",
+  "footer-menu-1-9-title": "Privatization deployment",
+  "footer-menu-2-title": "Sulotions",
+  "footer-menu-2-1-title": "Private Domain",
+  "footer-menu-2-2-title": "Customer Service",
+  "footer-menu-2-3-title": "Scalable",
+  "footer-menu-2-4-title": "Concersion",
+  "footer-menu-2-5-title": "Consumer Marketing",
+  "footer-menu-2-6-title": "Chain Stores",
+  "footer-menu-2-7-title": "Goverment",
+  "footer-menu-2-8-title": "Internet",
+  "footer-menu-3-title": "Resources",
+  "footer-menu-3-1-title": "Product",
```

Comment:
Maybe we can change to `Help Center`

### Intercom
![image](https://user-images.githubusercontent.com/6419514/150726056-7075d4fe-f4da-45c4-8799-e2ab7e27e348.png)


### Front
![image](https://user-images.githubusercontent.com/6419514/150726014-c0af8a14-e151-431d-b67f-cd2219af84d9.png)